src/sentry/static/sentry/app/utils/discover/eventView.tsx

```diff
@@ -918,7 +918,7 @@ class EventView {
     const environment = this.environment as string[];
 
     // generate event query
-    const eventQuery: EventQuery & LocationQuery = Object.assign(
+    const eventQuery = Object.assign(
```

Comment:
i think this change is more correct, eventQuery isn't `EventQuery & LocationQuery`, but instead we're casting the result from object.assign to `EventQuery & LocationQuery`